src/tests/stress/pedsim/Ped_util/IO.enc

```diff
@@ -0,0 +1,74 @@
+module IO
+local class File
+  var file : EMBED FILE* END
+  var fname : String
+  var mode : String
+  def init(fname : String, mode : String) : unit
+    this.fname = fname
+    this.mode = mode
+  end
+  def open(fin : String, mode : String) : unit
+    this.mode = if mode.equals("") then
+                  "w+r+a"
+                else
+                  mode
+                end
+    this.fname = fin
+    var fin = fin.data
+    var mChar = this.mode.data
+    this.file = EMBED (EMBED FILE* END)
+                  FILE *file = fopen(#{fin}, #{mChar});
+                  if (!file) {
+                    printf("Cannot open file, exiting.\n");
+                    exit(1);
+                  };
+                  file;
+
+                END
+  end
+  def close() : unit
+    var f = this.file
+    this.file = EMBED (EMBED FILE* END)
+                  if (!#{f}) {
```

Comment:
Added comment about that file and `XML.enc`, both should ideally be replaced with standard library stuff when that is available.